fili-core/src/test/groovy/com/yahoo/bard/webservice/util/DimensionStoreKeyUtilsSpec.groovy

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.util
+
+import com.yahoo.bard.webservice.config.BardFeatureFlag
+import spock.lang.Specification
+
+class DimensionStoreKeyUtilsSpec extends Specification {
+
+    def "Test getRowKey() with CASE_SENSITIVE_KEYS_ENABLED off"() {
+        expect:
+        DimensionStoreKeyUtils.getRowKey("FOO","1") == "foo_1_row_key"
```

Comment:
It would be good to explicitly set the flag to off to ensure that we're testing with the condition we want to test, rather than relying on default config. Not a requirement, but it would be nice.